go-sdk/pkg/sdk/events.go

```diff
@@ -0,0 +1,141 @@
+package sdk
```

Comment:
I've refactored a bit and I've moved out these functions from `keptn.go` to have it in one place and covered by basic unit tests. This logic can/should be greatly improved in a future PR